flink-kubernetes/src/test/java/org/apache/flink/kubernetes/kubeclient/Fabric8FlinkKubeClientTest.java

```diff
@@ -504,6 +550,7 @@ private KubernetesConfigMap buildTestingConfigMap() {
                         .withNewMetadata()
                         .withName(TESTING_CONFIG_MAP_NAME)
                         .withLabels(TESTING_LABELS)
+                        .withNamespace(NAMESPACE)
```

Comment:
I added this change as it makes it more handy when having the namespace also specified in the ConfigMap. See [KubernetesClientTestBase.mockConfigMapRequest](https://github.com/XComp/flink/blob/2e475a777ebebbbcee47eaafaa14c3ccf2724367/flink-kubernetes/src/test/java/org/apache/flink/kubernetes/KubernetesClientTestBase.java#L77). It's not really necessary since `NAMESPACE` is also available in that class. But I ran into the issue of the missing namespace first before discovering that the namespace is set through a static variable.